lms/views/email.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from h_pyramid_sentry import report_exception
+from pyramid.httpexceptions import HTTPFound
+from pyramid.view import view_config
+
+from lms.services import EmailUnsubscribeService
+from lms.services.exceptions import ExpiredJWTError, InvalidJWTError
+
+LOG = logging.getLogger(__name__)
+
+
+@view_config(
+    route_name="email.unsubscribe",
+    request_method="GET",
+    renderer="lms:templates/message.html.jinja2",
+)
+def unsubscribe(request):
+    """Unsubscribe the email and tag combination encoded in token."""
+    try:
+        request.find_service(EmailUnsubscribeService).unsubscribe(
+            request.matchdict["token"]
+        )
+    except (InvalidJWTError, ExpiredJWTError):
+        LOG.exception("Invalid unsubscribe token")
+        report_exception()
+        return {
+            "title": "Expired unsubscribe link",
+            "message": """
+                    <p>
+                        It looks like the unsubscribe link that you clicked on was invalid or had expired.
+                        Try clicking the unsubscribe link in a more recent email instead.
+                    </p>
+                    <p>
+                        If the problem persists, you can
+                         <a href="https://web.hypothes.is/get-help/?product=LMS_app" target="_blank" rel="noopener noreferrer">open a support ticket</a>
```

Comment:
I think you could just put this title and message HTML directly in an `unsubscribe_error.html.jinja2` template and have the Python function just `return {}`. It's nice to avoid having HTML strings in Python code, and also means you don't need to `assert` about HTML strings in the tests